now-or-later/Cargo.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "now-or-later"
+version = "1.0.0-beta.2"
```

Since this is a non-zebra crate, it should have its own version numbers:
```suggestion
version = "0.0.1"
```

---

Done.